manchester_traffic_offences/templates/plea/review.html

```diff
@@ -181,39 +188,39 @@ <h1 class="heading-xlarge">Confirm your plea{{ case.number_of_charges|pluralize
         <table class="review-table" id="your-money">
             <thead>
                 <tr>
-                    <th class="header">Your finances</th>
-                    <th class="change-link"><a href="{% url "plea_form_step" "your_money" %}" class="review">Change your finances</a></th>
+                    <th class="header">{% trans "Your finances" %}</th>
+                    <th class="change-link"><a href="{% url "plea_form_step" "your_money" %}" class="review">{% trans "Change your finances" %}</a></th>
                 </tr>
             </thead>
             <tbody>
                 <tr>
-                    <th class="label">You are</th>
+                    <th class="label">{% trans "You are" %}</th>
                     <td>{{ your_money.you_are }}</td>
                 </tr>
                 {% if your_money.you_are == "Employed" %}
                 <tr>
-                    <th class="label">Your Job</th>
+                    <th class="label">{% trans "Your Job" %}</th>
                     <td>{{ your_money.employed_your_job }}</td>
                 </tr>
                 <tr>
-                    <th class="label">You get paid</th>
+                    <th class="label">{% trans "You get paid" %}</th>
                     <td>{{ your_money.employed_take_home_pay_period }}</td>
                 </tr>
                 <tr>
-                    <th class="label">Amount</th>
+                    <th class="label">{% trans "Amount" %}</th>
                     <td>£{{ your_money.employed_take_home_pay_amount }}</td>
                 </tr>
                 {% endif %}
                 {% if your_money.you_are == "Self-employed" %}
                 <tr>
-                    <th class="label">Your Job</th>
+                    <th class="label">{% trans "Your Job" %}</th>
                     <td>{{ your_money.your_job }}</td>
                 </tr>
                 <tr>
-                    <th class="label">You get paid</th>
+                    <th class="label">{% trans "You get paid" %}</th>
                     <td>
-                        {% if your_money.self_employed_pay_period == "Self-employed other" %}
-                        Other
+                        {% if your_money.self_employed_pay_period == "self-employed other" %}
```

Comment:
Can you change "self-employed other" back to "Self-employed other" ?

@froddd noticed it and wanted me to credit him :)